pylzy/lzy/env/explorer/auto.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import List, Type, TypeVar
+
+from lzy.env.python.base import PackagesDict
+
+from .base import BaseExplorer, ModulePathsList, PackagesDict
+from .classify import ModuleClassifier
+from .search import VarsNamespace, get_transitive_namespace_dependencies
+from .packages import (
+    BrokenModules,
+    LocalPackage,
+    BasePackage,
+    PypiDistribution,
+    LocalDistribution
+)
+
+
+P = TypeVar('P', bound=BasePackage)
+
+
+@dataclass
+class AutoExplorer(BaseExplorer):
+    pypi_index_url: str
+    additional_pypi_packages: PackagesDict
+
+    def get_local_module_paths(self, namespace: VarsNamespace) -> ModulePathsList:
+        packages = self._get_packages(namespace, LocalPackage)
+
+        filtered: List[LocalPackage] = []
```

Comment:
what does it mean?

Reply:
Packages that were filtered by an additional_pypi_packages